.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Send Feedback
```

Oh, and don't forget this.

```suggestion
  - name: Contact Us
```

...or, in a more cheeky way...


```suggestion
  - name: Have something to say?
```

---

You can pick one you prefer and I will merge this, or I could do this tommorow.